idaes/power_generation/unit_models/boiler_heat_exchanger.py

```diff
@@ -128,47 +150,15 @@ def delta_temperature(b, t):
 
 
 @declare_process_block_class("BoilerHeatExchanger")
-class BoilerHeatExchangerData(UnitModelBlockData):
-    """
-    Standard Heat Exchanger Unit Model Class
-    """
-
-    CONFIG = ConfigBlock()
-    CONFIG.declare(
-        "dynamic",
-        ConfigValue(
-            domain=DefaultBool,
-            default=useDefault,
-            description="Dynamic model flag",
-            doc="""Indicates whether this model will be dynamic or not,
-**default** = useDefault.
-**Valid values:** {
-**useDefault** - get flag from parent (default = False),
-**True** - set as a dynamic model,
-**False** - set as a steady-state model.}""",
-        ),
-    )
-    CONFIG.declare(
-        "has_holdup",
-        ConfigValue(
-            default=useDefault,
-            domain=DefaultBool,
-            description="Holdup construction flag",
-            doc="""Indicates whether holdup terms should be constructed or not.
-Must be True if dynamic = True,
-**default** - False.
-**Valid values:** {
-**True** - construct holdup terms,
-**False** - do not construct holdup terms}""",
-        ),
-    )
+class BoilerHeatExchangerData(HeatExchangerData):
+    CONFIG = HeatExchangerData.CONFIG(implicit=True)
```

Comment:
Do we need implicit declaration here? We generally avoid that except for very specific circumstances (placeholders for other model's configs).

Reply:
Yes.  The heat exchanger lets you put the configuration for each side in under whatever you named it.  There are sub config blocks for each side.